app/components/chart/basics.js

```diff
@@ -188,6 +192,21 @@ var Basics = React.createClass({
     this.props.onSwitchToDaily();
   },
 
+  handleClickPrint: function(e) {
+    if (e) {
+      e.preventDefault();
+    }
+
+    if (this.props.pdf.url) {
+      const printWindow = window.open(this.props.pdf.url);
+      printWindow.focus();
+      printWindow.print();
+    }
+
+    // Send tracking metric
```

Comment:
maybe the prop name should change to reflect that is firing a metric?? just noting the comment

Reply:
done